### ISSUE TITLE
Quick accessibility review

### ISSUE THREAD

**Issue by richard-jones** [ImperialCollegeLondon/fair-data-repository#208]
Take a look at:
* The home page
* Search page
* A landing page

And flag any accessibility issues that may need to be resolved.

Imperial accessibility guidelines for reference: https://www.imperial.ac.uk/staff/tools-and-reference/web-guide/training-and-events/materials/accessibility/

# Acceptance Criteria

* A list of accessibility issues that may need to be addressed

**Comment by amdomanska** [comment]
Please find the document with findings attached .
[Imperial - Accessibility Review.pdf](https://github.com/user-attachments/files/19893068/Imperial.-.Accessibility.Review.pdf)

**Comment by npapantonis** [comment]
Thank you @amdomanska .  

The accessibility results don't seem too bad and all appear to be relatively easy fixes to resolve.  Unsure if the "Contrast" issues can/should be addressed by you on the branding ticket or if you think we should look into this?  

@cc-a the report highlights some issues on labels, headings etc. which look like quick fixes (said with absolutely no experience on Invenio code) although can we address these?

Thanks

**Comment by cc-a** [comment]
Some but unfortunately several (particularly the search ones) would involve overriding and working with fairly complex javascript components which is both fiddly and difficult to maintain longer term. Ideally those issues might be fixed in the upstream libraries.

**Comment by richard-jones** [comment]
@amdomanska can you see any quick wins or anything that we can do easily in the branding layer to address any of the accessibility shortfalls?  Assuming that we don't want to dive into the details of the base invenio implementation.

**Comment by amdomanska** [comment]
Majority of them would require overwriting at least templates and styles (as change in a template structure would most probably trigger styling issues). We could address contrast errors and focus indicators as these are purely scss issues - with the latter being definitely more impactful.

**Comment by npapantonis** [comment]
Thanks @amdomanska for the update.  I think if there are any quick wins then we should try to take them but also not make an adverse impact on the repo.  We are assigning the report you populated to our Accessibility person so they can review and guide us accordingly but we will mention none of these are easy fixes (if at all).  Maybe we can cover it for future plans and a disclaimer but will see what they advise.
